tests/pytorch/r1.9/colab.libsonnet

```diff
@@ -1,55 +0,0 @@
-local common = import 'common.libsonnet';
```

Comment:
Can you include the colab notebook tests in r1.11?

Reply:
added